### PR TITLE
Project stories index page

### PR DESCRIPTION
**Pull request by mybits**
As agreed we prefer to have scalable images and use object-fit across all browsers except from IE :(

### CODE REVIEW

**Comment by tomsabin** [comment]
I'm not sure if you have access to the original assets but it would be great to get some retina images here too

**Comment by tomsabin** [comment]
screen shot 2015-11-19 at 09 44 34
The images aren't loading for me when visiting "http://localhost:4567/project-stories/", however they load when visiting "http://localhost:4567/project-stories". We should prepend `/` before referencing the images so that we are using absolute path references.

**Comment by tomsabin** [comment]
I think we are using the wrong breakpoints to update the layout. I'd expect the transition to be smooth from the mobile view up to the maximum mobile container. However between 479px and 480px the image height snaps:



I think the image ratio should be maintained on the mobile view, which would mean the height of the tile increases as the width. It'll prevent us from having images cropped like below.

screen shot 2015-11-19 at 09 56 25

File: source/assets/stylesheets/styles/_project_story_large_tile.scss [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+.project-story-large-tile__image {
+  display: block;
+  height: 125px;
+  object-fit: cover;
+  width: 100%;
+
+  @include breakpoint($max-mobile-container) {
+    height: 180px;
+  }
```
Comment (tomsabin) [inline source/assets/stylesheets/styles/_project_story_large_tile.scss]: As mentioned in a [previous comment](https://github.com/unboxed/ubxd_web_refresh/pull/49#issuecomment-158009519), I think we should allow the images to maintain their ratios, so for the mobile view we would only need a 100% width block element, but from tablet upwards we should add the `object-fit` and a specified height to contain it.

File: source/assets/stylesheets/styles/_project_story_large_tile.scss [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  width: 100%;
+
+  &:hover {
+    background-color: $project-story-tile__hover-color;
+  }
+}
+
+.project-story-large-tile__image-container {
+  box-sizing: border-box;
+}
```
Comment (tomsabin) [inline source/assets/stylesheets/styles/_project_story_large_tile.scss]: This seems like an unnecessary class and additional `div` to the markup. I think we could get away with just:

``` CSS
.project-story-X-tile
  .project-story-X-tile__container
    img.project-story-X-tile__image
    .project-story-X-tile__content
       ...
```

File: source/assets/stylesheets/styles/_project_story_large_tile.scss [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  text-decoration: none;
+
+  @include breakpoint($max-mobile-container) {
+    min-height: 75px;
+  }
+
+  @include breakpoint($tablet-breakpoint) {
+    padding: 20px 6px 0;
+    width: 85%;
+    min-height: 170px;
```
Comment (tomsabin) [inline source/assets/stylesheets/styles/_project_story_large_tile.scss]: For tablet upwards I can understand the requirement for `min-height` as we have a fixed layout. But for mobile we could be more relaxed and allow the size of the text to determine its height - just ensuring that we have some breathing space at the bottom of the tile! See below:

screen shot 2015-11-19 at 10 13 24

(top needs more padding, bottom has too much space)

File: source/assets/stylesheets/styles/_project_story_large_tile.scss [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  }
+}
+
+.project-story-large-tile__client {
+  display: none;
```
Comment (tomsabin) [inline source/assets/stylesheets/styles/_project_story_large_tile.scss:55]: I quite like this but I think we should pass it through Graeme as well.

File: source/assets/stylesheets/styles/_project_story_large_tile.scss (hunk 1 above)
Comment (mybits) [inline source/assets/stylesheets/styles/_project_story_large_tile.scss]: Oh yes, the reason for adding so many heights was a way for fitting as much as possible content on mobile (hence smaller height on smaller resolutions). But you're right it doesn't scale well. 
We're gonna have new images so they will fit better, but I'm gonna take your advice and leave the images ratio as they are :)

File: source/assets/stylesheets/styles/_project_story_large_tile.scss (hunk 2 above)
Comment (mybits) [inline source/assets/stylesheets/styles/_project_story_large_tile.scss]: Yes, good point!

File: source/assets/stylesheets/styles/_project_story_large_tile.scss (hunk 3 above)
Comment (mybits) [inline source/assets/stylesheets/styles/_project_story_large_tile.scss]: Oh yeah, I wanted to make all tiles to be have same height. But you're right - it looks better when text determines it rather then it's fixed.

**Comment by mybits** [comment]
Thanks @tomsabin for the comments. I addressed them here: b7b83fd. Also spoke to Graeme about not having clients' names on mobile view and he confirmed it's ok.

**Comment by mybits** [comment]
Thanks @tomsabin for catching that error! Fixed in 81f6f50

**Comment by tomsabin** [comment]
LGTM :grapes:

**Pull request merged**